addons/mail/static/tests/new/message/message_tests.js

```diff
@@ -210,6 +210,65 @@ QUnit.module("mail", (hooks) => {
         );
     });
 
+    QUnit.test("Message background on edit mode", async function (assert) {
+        const pyEnv = await startServer();
+        const channelId = pyEnv["mail.channel"].create({
+            channel_type: "channel",
+            name: "channel1",
+        });
+        const [currentPartnerMessageId, otherPartnerMessageId] = pyEnv["mail.message"].create([
+            {
+                body: "Hello world",
+                res_id: channelId,
+                model: "mail.channel",
+                message_type: "comment",
+            },
+            {
+                body: "Goodbye world",
+                res_id: channelId,
+                model: "mail.channel",
+                message_type: "comment",
+                author_id: pyEnv["res.partner"].create({ name: "Bob" }),
+            },
+        ]);
+        const { click, openDiscuss } = await start({
+            discuss: {
+                context: {
+                    active_id: channelId,
+                },
+            },
+        });
+        await openDiscuss();
+        await click(
+            `.o-mail-message[data-message-id='${currentPartnerMessageId}'] i[aria-label='Edit']`
+        );
+        assert.strictEqual(
+            window
+                .getComputedStyle(
+                    document.querySelector(
+                        `.o-mail-message[data-message-id='${currentPartnerMessageId}'] .o-mail-message-body`
+                    )
+                )
+                .getPropertyValue("background-color"),
+            "rgb(227, 243, 231)",
+            "Message body of message in edition for current partner should be bg-success"
+        );
+        await click(
+            `.o-mail-message[data-message-id='${otherPartnerMessageId}'] i[aria-label='Edit']`
+        );
+        assert.strictEqual(
+            window
+                .getComputedStyle(
+                    document.querySelector(
+                        `.o-mail-message[data-message-id='${otherPartnerMessageId}'] .o-mail-message-body`
+                    )
+                )
+                .getPropertyValue("background-color"),
+            "rgb(237, 248, 250)",
+            "Message body of message in edition for other partner should be bg-info"
+        );
+    });
+
```

Comment:
No test for that.

Feel the need because it broke, but the underlying issue is template not properly reusing same styling for bubble.
No critical feature break with such a failed test, which is usually the case with poor CSS color style. We don't need QUnit tests for them.